src/tools/CMakeLists.txt

```diff
@@ -21,5 +21,11 @@ foreach(PROG xtp_map xtp_run xtp_tools xtp_parallel xtp_dump)
      install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROG}.man DESTINATION ${MAN}/man1 RENAME ${PROG}.1)
   endif (BUILD_MANPAGES)
   list(APPEND XTP_PROGS "${PROG}")
+
+  if(ENABLE_TESTING)
+    add_test(${PROG}Help ${PROG} --help)
```

Comment:
@baumeier: This is the cmake way to run test with `make test` (when build with `-DENABLE_TESTING=ON`).

Currently the test are just `PROGRAM --help`, but this is meant as a starting point.